### DISCUSSION TITLE
Weird Authentication Problem

### DISCUSSION THREAD

**Discussion by dakaloedu** [community/community#37050] [Other Feature Feedback, Questions, & Ideas]
When I try to sign up with my school email GitHub says I have already signed up, which I did not, and when I try to use forgot password method it says I have not registered with the email.  What could be the problem?

**Comment by MARYANNE67** [comment]
Try going into your school email and searching for a GitHub verification message. If not found, you could try searching for your GitHub account by email to see if it has been used to sign up previously.

**Reply by dakaloedu** [reply]
I've litteraly searched the word "github" and nothing appears, but my problem is very odd , I can't sign up nor sign in.

**Comment by airtower-luna** [comment]
You might want to [contact GitHub support](https://support.github.com/request) about that. They should be able to see what's going on. Maybe someone with a similar address added yours to their account, or the school messed up by reusing addresses? Anyway, good luck!

**Comment by rudransh-verma** [comment]
> When I try to sign up with my school email GitHub says I have already signed up, which I did not, and when I try to use forgot password method it says I have not registered with the email. What could be the problem?

Contact github support, I guess thats the only left option.Hopefully It would work.

**Comment by rudransh-verma** [comment]
Contact github support, I guess thats the only left option.Hopefully It would work.
